Library/Homebrew/linkage_checker.rb

```diff
@@ -3,12 +3,13 @@
 require "formula"
 
 class LinkageChecker
-  attr_reader :keg
+  attr_reader :keg, :formula
   attr_reader :brewed_dylibs, :system_dylibs, :broken_dylibs, :variable_dylibs
   attr_reader :undeclared_deps, :reverse_links
 
-  def initialize(keg)
+  def initialize(keg, formula=nil)
```

Comment:
The `=` lacks surrounding spaces. Would be nice to run RuboCop on this file to see if it flags any other style issues.